src/agent-page.js

```diff
@@ -64,9 +67,23 @@ const agentElements = {
     })
     return cells
   },
+
   onTrips: (num) => {
     return `<p>Traveling Today: ${num}</p>`
   },
+
+  renderMyCustomersPage: `
+    <section class='seach-page'>
+      <label class='search-label' for='customer-search'>Search Records:
+      <input class='customer-search' id='customer-search' role="search" type="search" name="search" placeholder="Customer Name">
+      </label>
+      <section class='results' id="results">
+      </section>
+    </section>
+    `,
+
+  renderError: `<p>User Does Not Exist</p>`
+
 }
 
 module.exports = agentElements;
```

Comment:
This is the CommonJS export syntax. I would look into using ES6 module syntax so that the two styles are not mixing in your app.